src/components/Button/Button.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from "react";
+import "./Button.css";
+
+class Button extends Component {
```

Comment:
Maybe `<input type="number">` would be better here - no need to handle incrementing/decrementing the value :)

Reply:
wanted to be like all the other cool kids on the playground with this :D 
also this is on purpose, since the app performance decreases with a higher number of jokes. 
This way the user will be 'discouraged' to click e.g. 50 times just to get 50 jokes. :)